src/targets/services/konnectorAlerts/createTriggerAt.js

```diff
@@ -1,14 +1,23 @@
 import { TRIGGER_DOCTYPE } from 'doctypes'
 import { logger } from 'ducks/konnectorAlerts'
-import { ONE_DAY } from 'ducks/recurrence/constants'
-import { getTriggerStates, fetchRelatedFuturAtTriggers } from './helpers'
-
-const dateInDays = (referenceDate, n) => {
-  return new Date(+new Date(referenceDate) + n * ONE_DAY)
-}
+import {
+  timeAhead,
+  getTriggerStates,
+  fetchRelatedFuturAtTriggers
+} from './helpers'
```

Comment:
I know this was already the case, but can you change the import to use alias instead of relative path?

Reply:
Sure, I'll give it a look.